tools/project_manager/edit_common.inc

```diff
@@ -94,6 +95,39 @@ function genre_list($genre)
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
+function charsuite_list($charsuites, $project_charsuites)
+{
+    $global_charsuites = CharSuites::get_enabled();
+    $project_charsuites = [];
+
+    $all_charsuites = [];
+    foreach($global_charsuites as $global_charsuite)
+    {
+        $all_charsuites[$global_charsuite->name] = $global_charsuite->title;
+    }
+    foreach($project_charsuites as $project_charsuite)
+    {
+        $all_charsuites[$project_charsuite->name] = $project_charsuite->title;
+    }
+    $existing_charsuites = [];
+    foreach($charsuites as $charsuite)
+    {
+        $existing_charsuites[$charsuite] = TRUE;
+    }
+
+    echo "<select name='charsuites[]' required multiple size='3'>\n";
+    foreach($all_charsuites as $k=>$v)
```

Comment:
We should sort `$all_charsuites` by value (title of the charsuite) so they are always presented in a consistent fashion to the user.